test/ui-test/testSuites/suite_status/tst_searchFlows/test.feature

```diff
@@ -0,0 +1,57 @@
+#******************************************************************************
+# Status.im
+#*****************************************************************************/
+#/**
+# * \file   test.feature
+# *
+# * \test   Status Desktop - Search flows
+# * \date   August 2022
+# **
+# *****************************************************************************/
+
+Feature: Search feature (ctrl+F)
+
+    As a user, I want to search the different chats and messages of the app
+
+    Covers the search flows
+
+    Background:
+        Given A first time user lands on the status desktop and generates new key
+        When user signs up with username tester123 and password TesTEr16843/!@00
+        Then the user lands on the signed in app
+        When user joins chat room search-automation-test-1
+        When user joins chat room search-automation-test-2
+        Then the user opens the community portal section
+        Then the user lands on the community portal section
+        When the user creates a community named myCommunity, with description My community description, intro Community Intro and outro Community Outro
+        Then the user lands on the community named myCommunity
+        When the admin creates a community channel named automation-community, with description My description with the method bottom_menu
+        Then the user lands on the channel named automation-community
+        # Go back to the portal so that we see if the search really redirects
+        Then the user opens the community portal section
+
+    Scenario: User can search for a community channel
+        When the user opens the search menu
+        When the user searches for automation
+        Then the search menu shows 3 results
```

Comment:
How we can ensure there are only 3 results?

Reply:
I tried to choose only channel names that are not used, that way there are no messages that would conflict. Let me know if you think it should be changed.

Reply:
I understand it but wondering how we can ensure it bc in some point maybe these names will exist.. Not sure how to deal with it unless we create a controlled test environment. WDYT?

Reply:
I guess the number of results is not that important in that scenario. The goal is mostly that we should the right channel and click on it, so maybe I should just remove that line?